app/stores/UsersStore.ts

```diff
@@ -159,6 +159,25 @@ export default class UsersStore extends BaseStore<User> {
     return res.data;
   };
 
+  @action
+  fetchDocumentUsers = async (params: {
```

Comment:
All these `fetchBlah` calls are super similar, it would be good to refactor and cleanup how this is done in stores if you care to in a separate PR.

Reply:
Similar in manner to `fetchDrafts`, `fetchArchived` etc. of `DocumentsStore`?